src/components/App.tsx

```diff
@@ -65,6 +67,9 @@ class App extends Component<Props> {
             <h1 style={{ fontWeight: "normal" }}>{t("efectivo")}</h1>
             <div>
               <ButtonDropdown
+                style={{
+                  width: 159
+                }}
```

Comment:
This was added because the currency ticker next to the language would extend over the dropdown box.